packages/fluentui/e2e/.eslintrc.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": ["plugin:@fluentui/eslint-plugin/react-northstar"],
-  "root": true
+  "root": true,
+  "rules": {
+    "import/no-default-export": "off"
```

Comment:
Please why it's "off" for e2e?

Reply:
`e2e` is not a part of distributed code so it's not important to fix things there.